### PR TITLE
[BACKEND] Implementing the Send Message Functionality

### LINKED ISSUE #419: [BACKEND] Implementing the Send Message Functionality
I will implement the send message functionality which can be used by different users of the system.

### PR DESCRIPTION
**Pull request by omerfdeniz**
## Type of change

- [X] New feature (non-breaking change which adds functionality)

# Description

I implemented the send message functionality which can be used by different users of the system.

# Test results

```
(djangoenv) (base) omer-mbp:backend omer$ python manage.py test API.tests.MessageTest
Creating test database for alias 'default'...
System check identified no issues (0 silenced).
...
----------------------------------------------------------------------
Ran 3 tests in 0.267s

OK
Destroying test database for alias 'default'...
```

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have checked my code and corrected any misspellings
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: ecommerce/backend/API/tests/test_message.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+from django.urls import reverse
+from ..models import User, Customer
+from ..views.message import *
+from ..utils.crypto import Crypto
+
+sender = None
+receiver = None
+# Class for testing the CRUD operations of the Message endpoint
+class MessageTest(TestCase):
+    def setUp(self):
```
Comment (bekir96) [inline ecommerce/backend/API/tests/test_message.py:12]: Can you increase test number to check serializer.is_valid() and not sender.id

**Review by umutoksuz — APPROVED** [review]
Looks good to me!

**Review by furkannane — APPROVED** [review]
Nicely done.

**Review by bekir96 — CHANGES_REQUESTED** [review]
test number should be increase for new error check

File: ecommerce/backend/API/tests/test_message.py (hunk 1 above)
Comment (omerfdeniz) [reply ecommerce/backend/API/tests/test_message.py:12]: added tests for both of them: `test_send_message_with_invalid_data()` and `test_send_message_without_login()`

Comment (bekir96) [reply ecommerce/backend/API/tests/test_message.py:12]: good work 👍

Comment (bekir96) [reply ecommerce/backend/API/tests/test_message.py:12]: can you take ss of new test results and also put to PR description

**Review by bekir96 — APPROVED** [review]
Good work, you can merge

**Pull request merged**